modules/apps/commerce/commerce-currency-web/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
 	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.4.0"
 	compileOnly project(":apps:account:account-api")
 	compileOnly project(":apps:application-list:application-list-api")
+	compileOnly project(":apps:commerce:commerce-api")
```

Comment:
why?

Reply:
Because I need to reach the CommerceAccountActionKeys